packages/connected-private-route/src/tests/index.test.tsx

```diff
@@ -97,7 +97,13 @@ describe('ConnectedPrivateRoute', () => {
       authenticated: false,
```

Comment:
Let us add a test that assures us that when the redirect path is something like `some-resource/23` which corresponds to a React Router route defined as having path="some-resource/:id" this component will work as expected i.e. it will redirect to `some-resource/23` and not something like `some-resource/:id`

Reply:
on it